cider/exec/plan/parser/SubstraitToAnalyzerExpr.cpp

```diff
@@ -37,8 +37,15 @@ bool getExprUpdatable(std::unordered_map<std::shared_ptr<Analyzer::Expr>, bool>
 }
 
 bool isStringFunction(const std::string& function_name) {
-  std::unordered_set<std::string> supportedStrFunctionSet{
-      "substring", "substr", "lower", "upper", "trim", "ltrim", "rtrim"};
+  std::unordered_set<std::string> supportedStrFunctionSet{"substring",
```

Comment:
```c++
static std::unordered_set<std::string> funcs = {"substring", ...};
```
1. declare it as a static variable
2. naming the variable as short as it.

Reply:
updated